core/kernel/notif.c

```diff
@@ -93,6 +93,7 @@ uint32_t notif_get_value(bool *value_valid, bool *value_pending)
 void notif_send_async(uint32_t value)
```

Comment:
typo in the commit message
s/has/as/

Reply:
thanks. i'll fix that when squashing fixup commits.